packages/zapp/console/src/components/Executions/Tables/NodeExecutionsTable.tsx

```diff
@@ -47,8 +80,8 @@ export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
 
   const columnStyles = useColumnStyles();
   // Memoizing columns so they won't be re-generated unless the styles change
-  const columns = React.useMemo(() => generateColumns(columnStyles), [columnStyles]);
-  const tableContext = React.useMemo(
+  const columns = useMemo(() => generateColumns(columnStyles), [columnStyles]);
+  const tableContext = useMemo(
     () => ({ columns, state: { selectedExecution, setSelectedExecution } }),
     [columns, selectedExecution, setSelectedExecution],
   );
```

Comment:
!!!
@james-union you forgot to delete <DetailsPanel> block starting on line 118 (sorry can't tag that line with comment) from this component!
Panel lives on the parent, having its duplicate here is not what we want. You may need to pass down additional props to control it though.

Reply:
As the return gets skinnier on DetailsPanel removal, you may want to consider placing `content` variable contents directly there

Reply:
done